charts/consul/templates/api-gateway-controller-deployment.yaml

```diff
@@ -243,11 +243,7 @@ spec:
           name: consul-data
```

Comment:
nit: you probably don't need `HOST_IP` env variable on this container anymore because it's not used anymore

Reply:
Thanks for catching that!